.travis.yml

```diff
@@ -9,5 +9,12 @@ env:
   global:
     - MAVEN_OPTS="-XX:MaxPermSize=2g -Xmx4g"
     - JAVA_OPTS="-XX:MaxPermSize=2g -Xmx4g"
+install:
+  - set -e
+  - travis_retry sudo apt-get install -y --fix-missing libmagic1 libmagic-dev
+  - gem install --no-ri --no-rdoc pdd
+  - gem install --no-ri --no-rdoc est
```

Comment:
@mkordas please add also `set +e` at the end of `install` (travis is broken and in some cases like builds on OSX it fails because of `set -e`), so if you add `set -e` you should add `set +e` at the end (see https://github.com/travis-ci/travis-ci/issues/5120)